packages/api-core/src/me/types.ts

```diff
@@ -6,28 +6,81 @@ export interface SocialLoginConnection {
 	service_user_id: string;
 }
 
+export interface UserMetaData {
+	links: Record< 'self' | 'help' | 'site' | 'flags', string >;
+	data: {
+		flags: {
+			active_flags: string[];
+		};
+	};
+}
+
 export interface User {
 	ID: number;
-	username: string;
-	display_name: string;
+	abtests: Record< string, string >;
+	atomic_site_count: number;
+	atomic_visible_site_count: number;
 	avatar_URL?: string;
-	language: string;
-	locale_variant: string;
-	logout_URL?: string;
+	date: string;
+	display_name: string;
 	email: string;
 	email_verified: boolean;
-	is_valid_google_apps_country: boolean;
+	had_hosting_trial: boolean;
+	has_jetpack_partner_access?: boolean;
 	has_unseen_notes: boolean;
+	i18n_empathy_mode: boolean;
+	is_google_domain_owner?: boolean;
+	is_subscription_only: boolean;
+	is_valid_google_apps_country: boolean;
+	jetpack_partner_types?: string[];
+	jetpack_site_count?: number;
+	jetpack_visible_site_count?: number;
+	language: string;
+	lasagna_jwt: string;
+	locale_variant: string;
+	logout_URL: string;
+	meta: UserMetaData;
+	newest_note_type: string;
+	phone_account: boolean;
+	primary_blog: number;
+	primary_blog_is_jetpack: boolean;
+	primary_blog_url: string;
+	profile_URL: string;
 	site_count: number;
-	visible_site_count: number;
-	meta: {
-		data: {
-			flags: {
-				active_flags: UserFlags[];
-			};
-		};
-	};
 	social_login_connections: SocialLoginConnection[];
+	use_fallback_for_incomplete_languages: boolean;
+	user_ip_country_code: string;
+	username: string;
+	verified: boolean;
+	visible_site_count: number;
+
+	/**
+	 * The bootstraped user's primary site slug.
+	 * @deprecated Use `primary_blog_url` instead.
+	 */
+	primarySiteSlug: string;
+
+	/**
+	 * The bootstraped user's locale slug, e.g. `es`.
+	 * @deprecated Use `language` instead.
+	 */
+	localeSlug: string;
+
+	/**
+	 * The bootstrapped user's locale variant, e.g. `es-mx`.
+	 * @deprecated Use `locale_variant` instead.
+	 */
+	localeVariant: string;
+
+	/**
+	 * The subkey for Subscription Management
+	 */
+	subscriptionManagementSubkey?: string;
+
+	/**
+	 * Whether the user is bootstrapped user
```

Comment:
Minor: But what exactly is a bootstrapped user, and ~~when is it used~~ why do we use it?

Reply:
It's used by the performance tracking in https://github.com/Automattic/wp-calypso/blob/9076f3bbfa36b71f4811b4e06d99b26e77299e1d/client/lib/performance-tracking/lib.js#L41. I'm not sure whether it's still useful.

Reply:
I mean it more in the sense of understanding the type. 

We have a user type, and then some of the parameters are specific to a bootstrapped user. What's the difference between a user and a bootstrapped user? Why isn't this a BootstrappedUser type? What the heck is a bootstrap user? Is it a user with less properties?.. .etc.. for me it's a bit confusing.